docs/telemetry.md

```diff
@@ -22,32 +92,163 @@ An example of the data collected is the following:
+          "key": "pillar_version",
+          "value": "8.0.33-25"
+        }
+      ]
+    }
+  ]
+}
+```
+
+The agent sends information about the database and metrics.
+
+| Key | Description |
+|---|---|
+| "id" | A generated Universally Unique Identifier (UUID) version 4 |
+| "createTime" | UNIX timestamp |
+| "instanceId" | The DB Host ID. The value can be taken from the `instanceId`, the `/usr/local/percona/telemetry_uuid` or generated as a UUID version 4 if the file is absent. |
+| "productFamily" | The value from the file path |
+| "metrics" | An array of key:value pairs collected from the Metrics file.
+
+The following operating system-level metrics are sent with each check:
+
+| Key | Description |
+|---|---|
+| "OS" | The name of the operating system |
+| "hardware_arch" | The type of processed used in the environment |
+| "deployment" | How the application was deployed. <br> The possible values could be "PACKAGE" or "DOCKER". |
+| "installed_packages" | A list of the installed Percona packages. <br> Other information is sent on a PostgreSQL installation. |
+
+The information includes the following:
+
+* Package name
+
+* Package version - the same format as Red Hat Enterprise Linux or Debian
+
+* Package repository - if possible
+
+The package names must fit the following pattern:
+
+* `percona-*`
+
+* `Percona-*`
+
+* `proxysql*`
+
+* `pmm`
+
+* `etcd*`
+
+* `haproxy`
+
+* `patroni`
+
+* `pg*`
+
+* `postgis`
+
+* `wal2json`
+
 ## Disable telemetry
 
-Telemetry is enabled by default. If you decide not to send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
+Percona software you just installed collects information about how you use it (telemetry) by default. Here's how to turn it off:
+
+### Within the First 24 Hours
+
+You have 24 hours to disable telemetry before any information is collected or sent.
+
+If you restart the database server after running the uninstall command, the telemetry component might automatically turn back on. This event happens because the server reloads settings during restart, and if those settings include instructions to use telemetry, then telemetry is enabled again.
+
+Use the following command to turn it off until the next restart:
 
-=== "Debian-derived distribution"
+```{.bash data-prompt=$}
+$ systemctl stop percona-telemetry-agent
+```
+
+Even after stopping the telemetry agent service, a different part of the software (DB component) creates a file related to telemetry every day and saves that file for seven days.
+
+To permanently disable telemetry, use this command:
+
+```{.bash data-prompt=$}
+$ systemctl disable percona-telemetry-agent
+```
+
+### To stop file creation
+
+You'll need to uninstall this component using the following command (without restarting the database server):
+
+```{.bash data-prompt="$"}
+$ UNINSTALL COMPONENT "file://component_percona_telemetry";
```

Comment:
I think it is worth to mention that this CMD shall be executed in PS console, because the previous commands were executed in Linux shell